packages/help-center/src/components/help-center-step-button.tsx

```diff
@@ -43,10 +45,14 @@ const HelpCenterButton: FC< HelpCenterButtonProps > = ( {
 	}
 
 	return (
-		<Button onClick={ openHelpCenter } className="step-wrapper__help-center-button">
-			{ helpCenterButtonText }
-		</Button>
+		<div className="step-wrapper__help-center-button-container">
+			{ translate( 'Questions? {{a}}Contact our site building team{{/a}}', {
```

Comment:
The text "Questions? Contact our site building team" makes sense only in the context of the DIFM flow, so I had added this as a flow property. If we move this string to the help center, we should probably check the flow name here before returning this string. 🙂

Reply:
Totally right. I wasn't sure about the urgency of the work on DIFM, so I went for the quick solution first. I'll check on the flow next.

Reply:
https://github.com/Automattic/wp-calypso/pull/100773